docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,640 @@
+                This field is not always present.
+                2. `feature` (REQUIRED, string). The feature the graph corresponds to. This is required, since all
+                relevant graphs are dependend on features.
+                3. `data` (list)
+                    1. `x_value` (REQUIRED, float). The $x$-value of the graph.
+                    2. `y_value` (REQUIRED, float). The $y$-value of the graph.
+
+### 3. Assessments
+
+There can be multiple assessments. For each assessment the following fields are present:
+
+1. `name` (REQUIRED). The name of the assessment.
+2. `date` (REQUIRED). The date at which the assessment is completed.
+3. `contents`. There can be multiple items in contents. For each item the following fields are present:
+
+    1. `question` (REQUIRED). A question.
+    2. `answer` (REQUIRED). An answer.
+    3. `remarks` (OPTIONAL). A field to put relevant discussion remarks in.
+    4. `authors`. There can be multiple names. For each name the following field is present.
+        1. `name` (OPTIONAL). The name of the author of the question.
+    5. `timestamp` (OPTIONAL). A timestamp of the date and time of the answer.
+
+## Example
+
+```yaml
+algorithm-registers:
+  - {id}                                                # Optional. Example: https://algoritmes.overheid.nl/nl/algoritme/45991543.
+language:
+  - {lang_0}                                            # Optional. Example nl.
+license: {licence}                                      # Required. Example: apache-2.0 or any license identifier from https://huggingface.co/docs/hub/repositories-licenses
+license_name: {licence_name}                            # Required. if license = other, specify an id for the licence. Example: 'my-license-1.0'
+license_link: {license_link}                            # Required. if license = other, specify "LICENSE" or "LICENSE.md" to link to a file of that name inside the repo, or a URL to a remote file.
+library_name: {library_name}                            # Optional. Example: keras or any library from https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/model-libraries.ts.
+tags:
+- {tag_0}                                               # Optional. Example: audio
+- {tag_1}                                               # Optional. Example: automatic-speech-recognition
+upl: {upl_uri}                                          # Optional. Example: https://standaarden.overheid.nl/owms/terms/AangifteVertrekBuitenland
+owners:
+- organization: {organization_name}                     # Required. Example: BZK
+  oin: {oin}                                            # Optional. Example: 00000001003214345000
+  name: {owner_name}                                    # Optional. Example: John Doe
+  email: {owner_email}                                  # Optional. Example: johndoe@email.com
+  role: {owner_role}                                    # Optional. Example: Data Scientist.
+
+model-index:
```

Comment:
Similar question. What do we think of adding a section `artifacts` where we can refer with a URI to a repository containing relevant artifact files for a model?

Reply:
And this too